lints.toml

```diff
@@ -0,0 +1,131 @@
+#
```

Comment:
it would be great to hide this file (along with `rustfmt.toml`) into `.maintain` dir. It should be possible via `Cargo.toml` config or some `cargo clippy .. -- --flag`.

Reply:
Yeah, maybe we simplify things and just have a .sh file in .maintain. I'm sure we can make the bash file look almost as pretty and then there's one less thing to compile/install.

Reply:
Resolved by just using a script in .maintain